install/README.md

```diff
@@ -26,21 +54,28 @@ to see if your system is capable of running Kata Containers.
 > - When choosing a stable release, replace all `master` occurrences in the URLs
 > with a `stable-x.y` version available on the [download server](http://download.opensuse.org/repositories/home:/katacontainers:/releases:/).
 
-### Distros
-
-* [CentOS](centos-installation-guide.md)
-* [Fedora](fedora-installation-guide.md)
-* [Red Hat](rhel-installation-guide.md)
-* [OpenSuse](opensuse-installation-guide.md)
-* [Ubuntu](ubuntu-installation-guide.md)
-* [SLES](sles-installation-guide.md)
+> **Notes on packages source verification**:
+> - The Kata packages hosted on the download server are signed with GPG to ensure integrity and authenticity.
+>
+> - The public key used to sign packages is available [at this link](https://github.com/kata-containers/tests/data/rpm-signkey.pub); the fingerprint is `9FDC0CB6 3708CF80 3696E2DC D0B37B82 6063F3ED`.
+>
+> - Only trust the signing key and fingerprint listed in the previous bullet point. Do not disable GPG checks,
+> otherwise packages source and authenticity is not guaranteed.
 
-### Cloud services
+#### Supported Distributions
+|Distro specific installation instructions                          | Versions        |
+|-------------------------------------------------------------------|-----------------|
+|[CentOS](centos-installation-guide.md)                             | 7               |
+|[Fedora](fedora-installation-guide.md)                             | 27, 28          |
+|[openSUSE](opensuse-installation-guide.md)                         | Leap (42.3)     |
+|[Red Hat Enterprise Linux (RHEL)](rhel-installation-guide.md)      | 7               |
+|[SUSE Linux Enterprise Server (SLES)](sles-installation-guide.md)  | SLES 12 SP3     |
+|[Ubuntu](ubuntu-installation-guide.md)                             | 16.04, 18.04    |
```

Comment:
We might want to add a [Clear Linux](https://clearlinux.org/documentation/clear-linux/tutorials/kata) entry here as well - we don't package for it on OBS, but Clear Linux packages Kata directly.

Reply:
@grahamwhaley I could add ClearLinux, but that table was made with the assumption that kata-manager or kata-doc-to-script can use the linked installation instructions.

So there are 2 possibilities: 
+ Add ClearLinux, and a note column, saying that installation cannot be scripted / automated for it
+ Have a new PR with a dedicated installation instructions pages, as done for other distros. However this would duplicate the information, something that @jodh-intel does not like :)

Reply:
Although I think @grahamwhaley has a good point about mentioning Clear Linux here, since that distro isn't actually mentioned at all, I think it's just about out of scope of this PR.

Let's land this and update for CLR on #270.